src/routes/product-detail/ProductDetailCard.tsx

```diff
@@ -29,34 +29,8 @@ export const ProductDetailCard = ({ product }: Props) => {
             <Flex />
           </Flex>
 
-          <Box maw={200}>
-            <Flex>
-              <Text w="100%" c="light-grey">
-                material
-              </Text>
-              <Text>cotton</Text>
-            </Flex>
-
-            <Flex>
-              <Text w="100%" c="light-grey">
-                manufacturer
-              </Text>
-              <Text>hering</Text>
-            </Flex>
-
-            <Flex>
-              <Text w="100%" c="light-grey">
-                price
-              </Text>
-              <Text>$23.99</Text>
-            </Flex>
-
-            <Flex>
-              <Text w="100%" c="light-grey">
-                discount
-              </Text>
-              <Text>$4.00</Text>
-            </Flex>
+          <Box>
+            <Text>{product.description}</Text>
           </Box>
```

Comment:
Uses a proper product description instead of the hard-coded product insights, which does not make sense for themes other than theStitch.